.github/workflows/ci.yml

```diff
@@ -85,7 +85,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: config
-      run: ./config no-asm no-makedepend enable-buildtest-c++ --strict-warnings -D_DEFAULT_SOURCE && perl configdata.pm --dump
+      run: ./config no-makedepend enable-buildtest-c++ --strict-warnings -D_DEFAULT_SOURCE && perl configdata.pm --dump
```

Comment:
This one does not seem to take that much time with no-asm. See for example the https://github.com/openssl/openssl/pull/13599 CI times. Can we please keep it?

Reply:
It is actually interesting why the legacy with no-asm takes so much more. Could it be the enable-crypto-mdebug or --debug combination with no-asm actually?

Reply:
ping @mattcaswell

Reply:
Ok, I added no-asm back to this line